app/javascript/components/widgets/weather/weather-effect.jsx

```diff
@@ -248,6 +248,56 @@ const Thunder = () => {
     </>
   );
 };
+// #endregion
+
+// cloud starts here
+const CloudMotionFrames = keyframes`
+  0% {
+    bottom: 100%
+  }
+  100% {
+    bottom: -12px;
+  }
+`;
+
+const Motion = styled.div`
+  position: absolute;
+  bottom: 100%;
+  pointer-events: none;
+  animation: 3s ease-in 1s infinite reverse both running slidein;
+`;
+
+const MovingCloud = styled(Motion)`
+  opacity: 90%;
+  background: white;
+  width: 200px;
+  height: 150px;
+  border-radius: 0 0 50px 50px;
+  background:
+  /*                radius                            position  / 2xradius*/ radial-gradient(
+        35px 30px,
+        grey 98%,
+        transparent 100%
+      )
+      20% 30% /70px 60px,
+    radial-gradient(50px 45px, grey 98%, transparent 100%) 50% 50% /100px 90px,
+    radial-gradient(50px, grey 98%, transparent 100%) 100% 100%/100px 100px,
+    radial-gradient(40px, grey 98%, transparent 100%) 0 100% /80px 80px,
```

Comment:
Indentation looks off here. :thinking: maybe we need something that checks/formats css strings? Not in this PR. Just thinking out loud.